### DISCUSSION TITLE
chart and backpack/permissionmanager

### DISCUSSION THREAD

**Discussion by soluzione-software** [Laravel-Backpack/community-forum#1040] [Q&A (Help)]
i'm using laravel backpack v. 6 and backpack pro.
i'm using "backpack/permissionmanager": "^7.1",
in my dashboard i have included a chart
if i'm logged with administrator the chart is displayed correctly
if i'm not logged with administrator the chart is not displayed (the spinner load infinitely);
it seems it do a redirect 302 to login

can someone help me

**Comment by jcastroa87** [comment]
Can you please your code to check if this is something about permissions.

Did you receive any console error? or any server error log?

Cheers.

**Comment by soluzione-software** [comment]


**Reply by jcastroa87** [reply]
Can you format the code please, and share the full error console and a screenshot of it, thanks.

**Comment by soluzione-software** [comment]
thanks @jcastroa87 .
i solve.
the problem was this:
php artisan backpack:chart WeeklyUsers
create route in a group when a middleware block all users and can only access to administrator

thanks
bye

**Comment by jcastroa87** [comment]
Great! so i will close the issue, but please feel free to re-open or create a new one if needed.

Cheers.
